typescript/cli/src/commands/registry.ts

```diff
@@ -1,10 +1,13 @@
-import { CommandModule } from 'yargs';
+import { CommandModule, Options } from 'yargs';
 
+import { createAgentConfig } from '../config/agent.js';
 import { CommandModuleWithContext } from '../context/types.js';
-import { log, logBlue, logGray, logTable } from '../logger.js';
+import { log, logBlue, logGray, logRed, logTable } from '../logger.js';
 
-const ChainTypes = ['mainnet', 'testnet'];
-type ChainType = (typeof ChainTypes)[number];
+import { outputFileCommandOption } from './options.js';
+
+export const ChainTypes = ['mainnet', 'testnet'];
+export type ChainType = (typeof ChainTypes)[number];
```

Comment:
This is a bigger question than you signed up for with this PR but do we definitely still need this mainnet/testnet differentiation? The direction we've been going with other tooling is to just treat chains equally. Feel free to punt on this

Reply:
@daniel-savu do the agents rely on this differentiation?

Reply:
I'm not sure what this type is used for, can you share some context?

Reply:
fwiw the agent configs don't have different schemas between testnet/mainnet, just different values for the same config items

Reply:
This type will be used to decide whether to generate the config for testnet or mainnet chains

Reply:
as far as the agents are concerned, it doesn't matter whether the chain is a mainnet or testnet